### ISSUE TITLE
[Questions] Replace with your question

### ISSUE THREAD

**Issue by dinhthengu** [arichorn/uYouPlusExtra#225]
### Have you read the FAQ?

- [X] Yes, I did read the FAQ

### Is there an existing issue/question for this?

- [X] I have searched the existing issues

### Do you think this is a bug?

❌ No, I don't think this is a bug. I will explain below

### My question

Dear every one
I wanna appear logo YouTube premium, i have set on setting but it dont work, please help me. Many thanks! 
![IMG_6831](https://github.com/arichorn/uYouPlusExtra/assets/151049281/cf767c7f-263e-4e15-94aa-d06776c4e9a5)

**Comment by Shaukariya** [comment]
Which version of the app are you using? I upgraded to the latest version today and it's working.

**Comment by dinhnhu291** [comment]
> Which version of the app are you using? I upgraded to the latest version today and it's working.
I upgraded to the latest version today and it's not working.
i install with AltStore

**Comment by dinhthengu** [comment]
> Which version of the app are you using? I upgraded to the latest version today and it's working.

I have just update latest version, but this not working

**Comment by Lazycaker** [comment]
This issue already has been reported.
https://github.com/arichorn/uYouPlusExtra/issues/207

**Comment by ruchirtrehan** [comment]
Doesn't work for me either.

**Comment by ballisticempathy** [comment]
> That’s odd? It should work? ![IMG_2486](https://user-images.githubusercontent.com/78001398/284485235-57b9c256-b6de-4170-801d-c72e477ff1f8.png)

I've tried everything from messing around with different settings to clean installing it, it did show up once but after an app restart it stopped and hasn't worked since.

**Comment by yodaluca23** [comment]
> I've tried everything from messing around with different settings to clean installing it, it did show up once but after an app restart it stopped and hasn't worked since. 

Question was that app restart the first one after you had reinstalled it?

I'm asking because my theory is that YouTube applies the A/B flags (or that they take an app restart to fully show up) of each user after the first app restart after they've logged in.

**Comment by dinhthengu** [comment]
> That’s odd? It should work? ![IMG_2486](https://user-images.githubusercontent.com/78001398/284485235-57b9c256-b6de-4170-801d-c72e477ff1f8.png)

Can you show me the setting? Please

**Comment by ballisticempathy** [comment]
> > I've tried everything from messing around with different settings to clean installing it, it did show up once but after an app restart it stopped and hasn't worked since.
> 
> Question was that app restart the first one after you had reinstalled it?
> 
> I'm asking because my theory is that YouTube applies the A/B flags (or that they take an app restart to fully show up) of each user after the first app restart after they've logged in.

No, I reinstalled it much later, it worked when I turned off YTModernUI & a few other options and restarted but after another restart, the premium logo stopped showing (no settings changed) I've tried messing with settings since and had no luck getting it to show again.

**Comment by dinhthengu** [comment]
> > > I've tried everything from messing around with different settings to clean installing it, it did show up once but after an app restart it stopped and hasn't worked since.
> > 
> > 
> > Question was that app restart the first one after you had reinstalled it?
> > I'm asking because my theory is that YouTube applies the A/B flags (or that they take an app restart to fully show up) of each user after the first app restart after they've logged in.
> 
> No, I reinstalled it much later, it worked when I turned off YTModernUI & a few other options and restarted but after another restart, the premium logo stopped showing (no settings changed) I've tried messing with settings since and had no luck getting it to show again.

And now, you only see YouTube logo?

**Comment by ballisticempathy** [comment]
> > > > I've tried everything from messing around with different settings to clean installing it, it did show up once but after an app restart it stopped and hasn't worked since.
> > > 
> > > 
> > > Question was that app restart the first one after you had reinstalled it?
> > > I'm asking because my theory is that YouTube applies the A/B flags (or that they take an app restart to fully show up) of each user after the first app restart after they've logged in.
> > 
> > 
> > No, I reinstalled it much later, it worked when I turned off YTModernUI & a few other options and restarted but after another restart, the premium logo stopped showing (no settings changed) I've tried messing with settings since and had no luck getting it to show again.
> 
> And now, you only see YouTube logo?

Yeah
